livedata/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -46,9 +46,7 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.cancelChildren
 import kotlinx.coroutines.delay
 import java.lang.Thread.sleep
-import java.util.Date
-import java.util.InputMismatchException
-import java.util.UUID
+import java.util.*
```

Comment:
my editor has the same issue

Reply:
It can be configured into AS Setting "Editor > Code Style > Kotlin" and only allow "Use single name import" and remove `java.util` from "package to use import with '*'" section